templates/publisher/market.html

```diff
@@ -234,7 +234,7 @@ <h1 class="u-float--left p-heading--three">{% if display_title %}{{ display_titl
             <div class="col-8">
               <div class="p-form-validation u-no-margin {% if field_errors and field_errors['website'] %}is-error{% endif %}">
                 <div class="p-form-validation__field">
-                  <input class="p-form-validation__input" type="url" name="website" value="{{ website }}" maxlength="256"/>
+                  <input class="p-form-validation__input" type="url" name="website" value="{{ website if website is not none else '' }}" maxlength="256"/>
```

Comment:
@nottrobin 

I'm not sure if it's the best place to fix that.
I considered doing it on python side in views, but it means doing it in even more places (separate views for get/post, etc). So maybe we should fix it in out API layer? Or maybe it's an API bug?

Reply:
I think this is good, but we should discuss with Toto about a better global solution